packages/frontend/src/app/layout.tsx

```diff
@@ -17,7 +19,12 @@ export default function RootLayout({
 }>) {
   return (
     <html lang="en">
-      <body className={cn('min-h-screen bg-background font-sans antialiased', fontSans.variable)}>{children}</body>
+      <body className={cn('min-h-screen bg-background font-sans antialiased', fontSans.variable)}>
```

Comment:
the body should be displayed as flex, with header stickeed to the top, {children} are wrapped in a <main> element, and have a flex grow, while footer element should always stick to the bottom